client/src/components/LoginSession/LoginSession.js

```diff
@@ -25,10 +26,12 @@ function LoginSession(props) {
         props.logout();
     }
 
-    const created = moment(props.Created);
-    const createdElapsed = moment.utc(props.Created).fromNow();
-    const lastAccessed = moment(props.LastAccessed);
-    const lastAccessedElapsed = moment.utc(props.LastAccessed).fromNow();
+    moment.locale(ss.i18n.currentLocale);
+    const format = 'L LT';
```

Comment:
When I have values that never change no matter how the component is rendered, I tend to define them at top of the file in the top context.

It's a way of illustrating that this is a "true" constant that is not affected by the execution of the code. It also has a very small performance boost since you are only defining this value once rather than for each instance of the component.

That's just a nice to have.

Reply:
Updated